src/components/learning-module/SideBar.js

```diff
@@ -45,9 +45,12 @@ function SideBar(props) {
           className={'sidebar ' + animationClass}
           onClick={stopPropagation}
           onAnimationEnd={onAnimationEnd}>
-          <div className="go-back-summary" onClick={onClickGoBackSummary}>
-            <img src={require('./images/arrow-left.svg')} alt="Arrow Left Icon" />
-            <p>{'Back To Summary'}</p>
+          <div className="sidebar-controls">
+            <div className="go-back-summary" onClick={onClickGoBackSummary}>
+              <img src={require('./images/arrow-left.svg')} alt="Arrow Left Icon" />
+              <p>{'Back To Summary'}</p>
```

Comment:
No need for {'text'} syntax for strings. Its just HTML syntax:
```<p>Back To Summary</p>```

Reply:
Lol that was actually something that whoever created this component used ;)

Reply:
Changing now however.